### PR TITLE
feat: update finalize script for ERC20 payouts

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- Builds on top of merkle implementation
- Updated fields in GrantPayout
  - added fk to Token table
  - added conversion_rate 
  - added conversion_date

- Updated fields in ClrMatch 
  - added fk to Token table
  - added token_amount 
  - updated amount to include help_text

- Updates payout script to ensure the above fields are taken into account as oppsed to hardcoding in DAI  

This needs to be run to map every  
- CLRMatch token to DAI 
- CLRMatch token_amount to the amount (cause DAI == USD) 
- GrantPayout token to DAI 

```python
from economy.models import Token
from grants.models import *

dai_token = Token.objects.get(pk=283)
clr_matches = CLRMatch.objects.all()
grant_payouts = GrantPayout.objects.all()

for clr_match in clr_matches:
    clr_match.token = dai_token
    clr_match.token_amount = clr_match.amount # This is DAI 
    clr_match.save()

for grant_payout in grant_payouts:
    grant_payout.token = dai_token
    grant_payout.save()
```
##### Refers/Fixes

- fixes #https://github.com/gitcoinco/web/issues/10120
- refs https://github.com/gitcoinco/web/issues/10116

##### Testing

Locally tested

### CODE REVIEW

File: app/assets/v2/js/grants/matching_funds.js [hunk 1]
```diff
@@ -46,8 +48,9 @@ Vue.mixin({
                 e.claim_start_date = e.claim_start_date ? moment(e.claim_start_date).format('MMM D') : null;
                 e.claim_end_date = e.claim_end_date ? moment(e.claim_end_date).format('MMM D, Y') : null;
               });
-              const claimData = await vm.checkClaimStatus(m, grant.admin_address);
 
+              // TODO: HAS TO BE UPDATED
```
Comment (thelostone-mc) [inline app/assets/v2/js/grants/matching_funds.js:52]: @gravityblast This would need to be updated

**Comment by thelostone-mc** [comment]
Note: This should be merged only after clawbacks have happened for earlier rounds

File: app/assets/v2/js/grants/matching_funds.js [hunk 2]
```diff
@@ -62,69 +66,60 @@ Vue.mixin({
...
+
+    async checkClaimStatus(match, adminAddress) {
       const contractAddress = match.grant_payout.contract_address;
       const txHash = match.claim_tx;
 
       let status = 'not-found';
       let timestamp = null;
 
-      web3 = new Web3(`wss://mainnet.infura.io/ws/v3/${document.contxt.INFURA_V3_PROJECT_ID}`);
+      web3 = new Web3(`wss://rinkeby.infura.io/ws/v3/${document.contxt.INFURA_V3_PROJECT_ID}`);
```
Comment (gravityblast) [inline app/assets/v2/js/grants/matching_funds.js:85]: @thelostone-mc I think we need to change it back to mainnet right?

**Review by gravityblast — APPROVED** [review]
LGTM!

just added one comment about using mainnet instead of rinkeby in the infura url

**Comment by schultztimothy** [comment]
Very cool 🚀 🚀 🚀 

I pulled it down and was going to run the tests to wrap my head around it all but was running into a few errors. [This](https://github.com/gitcoinco/web/compare/erc20-support...erc20-support-tests?expand=1 ) seemed to fix the import errors, but still running into a few failing tests 🤷

File: app/assets/v2/js/grants/matching_funds.js (hunk 2 above)
Comment (thelostone-mc) [reply app/assets/v2/js/grants/matching_funds.js:85]:
```suggestion
      web3 = new Web3(`wss://mainnet.infura.io/ws/v3/${document.contxt.INFURA_V3_PROJECT_ID}`);
```

**Comment by thelostone-mc** [comment]
Sweet! Updated your comment @gravityblast ! 
@schultztimothy got your changes merged in as well <3

**Pull request merged**